src/main/webapp/app/entities/participation/participation.component.html

```diff
@@ -12,127 +12,196 @@ <h2>
-                                class="btn btn-info btn-sm mr-1"
-                            >
-                                <fa-icon [icon]="'file-powerpoint'"></fa-icon>
-                                <span class="d-none d-md-inline" jhiTranslate="artemisApp.participation.addPresentation.label">Add presentation</span>
-                            </button>
-                            <button
-                                *ngIf="this.presentationScoreEnabled && participation.presentationScore == 1"
-                                (click)="removePresentation(participation)"
-                                class="btn btn-danger btn-sm mr-1"
-                            >
-                                <fa-icon [icon]="'file-powerpoint'"></fa-icon>
-                                <span class="d-none d-md-inline" jhiTranslate="artemisApp.participation.removePresentation.label">Remove presentation</span>
-                            </button>
+                    </ng-template>
+                </ngx-datatable-column>
+
+                <ngx-datatable-column prop="repositoryUrl" *ngIf="exercise?.type == PROGRAMMING" [minWidth]="120">
+                    <ng-template ngx-datatable-header-template>
+                        <span class="datatable-header-cell-wrapper" (click)="controls.onSort('repositoryUrl')">
+                            <span class="datatable-header-cell-label bold sortable" jhiTranslate="artemisApp.participation.repositoryUrl">
+                                Repository Url
+                            </span>
+                            <fa-icon [icon]="controls.iconForSortPropField('repositoryUrl')"></fa-icon>
+                        </span>
+                    </ng-template>
+                    <ng-template ngx-datatable-cell-template let-value="value">
+                        <span *ngIf="value != null">
+                            <a href="{{ value }}" target="_blank">Repository Link</a>
+                        </span>
+                    </ng-template>
+                </ngx-datatable-column>
+
+                <ngx-datatable-column prop="buildPlanId" *ngIf="exercise?.type == PROGRAMMING" [minWidth]="100">
+                    <ng-template ngx-datatable-header-template>
+                        <span class="datatable-header-cell-wrapper" (click)="controls.onSort('buildPlanId')">
+                            <span class="datatable-header-cell-label bold sortable" jhiTranslate="artemisApp.participation.buildPlanId">
+                                Build Plan Id
+                            </span>
+                            <fa-icon [icon]="controls.iconForSortPropField('buildPlanId')"></fa-icon>
+                        </span>
+                    </ng-template>
+                    <ng-template ngx-datatable-cell-template let-value="value">
+                        <span *ngIf="value != null">
+                            <a href="https://bamboobruegge.in.tum.de/browse/{{ value }}" target="_blank">
```

Comment:
I would suggest to not use `target="_blank"`. Here is a good read and some better ways to solve it https://medium.com/sedeo/how-to-fix-target-blank-a-security-and-performance-issue-in-web-pages-2118eba1ce2f

Reply:
I'm aware of the pitfalls of using `target="_blank"` and I agree that it's not ideal but I don't think fixing it should be part of this PR since it wasn't newly introduced here and our codebase has 33 such usages. If we want to address this, it should be a separate issue only pertaining to refactoring this anti-pattern.

Reply:
Can you at least add `rel=”noopener noreferrer"` in this case? This should be already enough

Reply:
Fair enough, I added it.